### PR TITLE
chore(release): publish packages

### PR DESCRIPTION
**Pull request by spydon**
- ubuntu_localizations@0.4.0
 - ubuntu_widgets@0.6.0
 - ubuntu_lints@0.3.1
 - ubuntu_test@0.1.0-beta.10

### CODE REVIEW

File: packages/ubuntu_widgets/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+## 0.6.0
```
Comment (d-loose) [inline packages/ubuntu_widgets/CHANGELOG.md:1]: Empty changelog here

Comment (spydon) [reply packages/ubuntu_widgets/CHANGELOG.md:1]: It's because all commits were chores, I'll add  manual entries

Comment (d-loose) [reply packages/ubuntu_widgets/CHANGELOG.md:1]: Thanks, same for localizations and lints

Comment (spydon) [reply packages/ubuntu_widgets/CHANGELOG.md:1]: Fixed!

**Review by d-loose — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,43 @@
...
+
+- Bump Flutter to `3.22.1`
+- 
+#### `ubuntu_widgets` - `v0.6.0`
+
+- Bump Flutter to `3.22.1`
+
+#### `ubuntu_lints` - `v0.3.1`
+
+- Bump ubuntu_lints to `4.0.0`
```
Comment (CarlosNihelton) [inline CHANGELOG.md:41]: I suspect you meant "Bump **flutter_lints** to `4.0.0.`", didn't you?

Comment (spydon) [reply CHANGELOG.md:41]: I did indeed, noticed it directly after I published 😄

**Pull request merged**